types/knockout/index.d.ts

```diff
@@ -557,14 +659,35 @@ interface KnockoutStatic {
     toJSON(viewModel: any, replacer?: Function, space?: any): string;
 
     toJS(viewModel: any): any;
-
+    /**
+     * Determine if argument is an observable. Returns true for observables, observable arrays, and all computed observables.
+     * @param instance Object to be checked
+     */
     isObservable(instance: any): instance is KnockoutObservable<any>;
+    /**
+     * Determine if argument is an observable. Returns true for observables, observable arrays, and all computed observables.
+     * @param instance Object to be checked
+     */
     isObservable<T>(instance: KnockoutObservable<T> | T): instance is KnockoutObservable<T>;
-
+    /**
+     * Determine if argument is an observable. Returns true for observables, observable arrays, and all computed observables.
```

Comment:
This should read as "and *writable* computed observables", not "all computed observables".

Reply:
done